src/OSPSuite.R/Services/ContainerTask.cs

```diff
@@ -115,6 +136,18 @@ public IParameter[] AllParametersForSensitivityAnalysisMatching(ISimulation simu
 
       public string[] AllStateVariableParameterPathsIn(IModelCoreSimulation simulation) => AllStateVariableParameterPathsIn(simulation?.Model?.Root);
 
+      public string[] BaseUnitNamesFromPath(IModelCoreSimulation simulation, string path) =>
+         AllQuantitiesMatching(simulation, path).Select(x => x.BaseUnitName()).ToArray();
+
+      public string[] DimensionNamesFromPath(IModelCoreSimulation simulation, string path) => 
+         AllQuantitiesMatching(simulation, path).Select(x => x.DimensionName()).ToArray();
+
+      public bool[] IsExplicitFormulasFromPath(IModelCoreSimulation simulation, string path) => 
```

Comment:
I am wondering how this will be used to be honest. 
Also I renamed add added a `s` to be plural again